app/src/main/java/com/framgia/musixmatch/data/source/AlbumDataSource.java

```diff
@@ -0,0 +1,11 @@
+package com.framgia.musixmatch.data.source;
+
+import com.framgia.musixmatch.data.model.Album;
+
+import java.util.List;
+
+public interface AlbumDataSource {
+    interface LocalDataSource {
```

Comment:
```suggestion
    interface Local {
```
em đặt tên thế này đựoc rồi
gọi AlbumDataSource. LocalDataSource 
có vẻ bị duplicate